src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Javadoc header comments are critical for interfaces. Else no one will know how to use the API!